test/kb_ea_utils_server_test.py

```diff
@@ -327,6 +327,87 @@ def getPairedEndLibInfo(self, read_lib_basename, lib_i=0):
         return new_obj_info
 
 
+    def getPairedEndInterleavedLibInfo(self, read_lib_basename, lib_i=0):
+        if hasattr(self.__class__, 'pairedEndLibInfo_list'):
+            try:
+                info = self.__class__.pairedEndLibInfo_list[lib_i]
+                name = self.__class__.pairedEndLibName_list[lib_i]
+                if info != None:
+                    if name != read_lib_basename:
+                        self.__class__.singleEndLibInfo_list[lib_i] = None
+                        self.__class__.singleEndLibName_list[lib_i] = None
+                    else:
+                        return info
+            except:
+                pass
```

Comment:
I used existing functions as a skeleton. Made this similar to other functions getSingleEndLibInfo(),  getPairedEndLibInfo().